lib/generators/stimulus_reflex/templates/esbuild.config.js.tt

```diff
@@ -0,0 +1,13 @@
+const path = require('path')
+const rails = require('esbuild-rails')
+
+require('esbuild')
+  .build({
+    entryPoints: ['application.js'],
+    bundle: true,
+    outdir: path.join(process.cwd(), 'app/assets/builds'),
+    absWorkingDir: path.join(process.cwd(), 'app/javascript'),
```

Comment:
The path `'app/javascript'` needs to be dynamic based on the selected location

Reply:
Good catch! Seems like this `copy_file` might need to become `template`.

I'm on it. 👍